lib/uffizzi/cli/cluster.rb

```diff
@@ -104,9 +103,9 @@ def handle_list_command(project_slug)
       end
     end
 
-    def handle_create_command(project_slug)
+    def handle_create_command(project_slug, command_args)
       Uffizzi.ui.disable_stdout if Uffizzi.ui.output_format
-      cluster_name = options[:name] || ClusterService.generate_name
+      cluster_name = command_args[:name] || ClusterService.generate_name
```

Comment:
Let's change the generate_name method to use faker.internet.domain_word like [here](https://github.com/UffizziCloud/uffizzi_platform/pull/1209/files#diff-9c997fbb226495c9613e9ca45f0d0f10f75d98aa45c038d84f90192a7f03659fR16)